nixos/modules/system/etc/etc-activation.nix

```diff
@@ -15,6 +15,7 @@
       system.activationScripts.etc = lib.stringAfter [
         "users"
         "groups"
+        "specialfs"
```

Comment:
This is needed so that we have /proc set up, since the activation script for the etc overlay needs to query the kernel for mounts.